src/upload-tests.proj

```diff
@@ -102,10 +102,21 @@
     </ItemGroup>
   </Target>
 
+  <Target Name="CopyTestILCToTestHostDir" Condition="'$(BuildingUAPAOTVertical)' == 'true'">
```

Comment:
I would much prefer we pull it down locally instead of pulling from a network share. We want to maintain consistency between local and helix runs as well as we want to maintain the ability to work offline after a sync of external has been done.